modules/menu/menu.c

```diff
@@ -793,6 +793,19 @@ static void ua_event_handler(struct ua *ua, enum ua_event ev,
 		menu_selcall(call);
 		break;
 
+	case UA_EVENT_CALL_REDIRECT:
+		val = pl_null;
+		if (!re_regex(prm, strlen(prm), "sip:"))
+			pl_set_str(&val, "invite");
+
+		(void)menu_param_decode(prm, "method", &val);
```

Comment:
Is `;method=invite` a possible parameter of the SIP 302 Contact?
Like for the SIP REFER?

Reply:
No it is not.  I was not sure why that code was there.  Will try to simplify.